tidb-cloud/import-sample-data.md

```diff
@@ -48,7 +46,13 @@ This document describes how to import sample data into TiDB Cloud via the UI. Th
 
 4. Click **Import**.
 
-    The data import process will take 5 to 10 minutes. When the data import progress bar shows **Success**, you have successfully imported the sample data and the database schema to your database in TiDB Cloud.
+    A warning message about the database resource consumption is displayed.
+
+5. Click **Confirm**.
+
+    TiDB Cloud starts validating whether it can access the sample data in the specified bucket URL. After the validation is completed and successful, the import task starts automatically. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).
```

Comment:
```suggestion
    TiDB Cloud starts validating whether it can access the sample data in the specified bucket URL. After the validation is completed and successful, the import task starts automatically.
```